src/replication.c

```diff
@@ -3237,10 +3237,10 @@ void setupMainConnForPsync(connection *conn) {
     if (psync_result == PSYNC_WAIT_REPLY) return; /* Try again later... */
 
     if (psync_result == PSYNC_CONTINUE) {
-        serverLog(LL_NOTICE, "Primary <-> REPLICA sync: Primary accepted a Partial Resynchronization%s",
```

Comment:
Changing this because in other logs we are using PRIMARY <-> REPLICA everywhere in this file.